src/components/bookmark-button/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { RemoveBookmarkIcon, AddBookmarkIcon } from './icons'
+import s from './bookmark-button.module.css'
+import useAuthentication from 'hooks/use-authentication'
+
+/**
+ * TODO
+ * - create a 'plugged' in HOC that can render modals, update bookmark state etc.
+ * This HOC will be used by this button and the one in tutorial meta
+ */
+
+interface BookmarkButtonProps {
+	isBookmarked: boolean
+}
+
+export default function BookmarkButton({ isBookmarked }: BookmarkButtonProps) {
+	const { isAuthEnabled } = useAuthentication()
+	// NOTE! - hiding this component from prod until auth is enabled
+	if (!isAuthEnabled) {
+		return null
+	}
+	const helpText = isBookmarked ? `Remove bookmark` : `Add bookmark`
+	return (
+		<button
+			aria-pressed={!!isBookmarked}
```

Comment:
Since `isBookmarked` is already a `boolean`, no need to `!!` here!

```suggestion
			aria-pressed={isBookmarked}
```